.circleci/config.yml

```diff
@@ -48,7 +48,38 @@ jobs:
           command: docker run --rm --tty --interactive application:$CIRCLE_SHA1 python --version | grep "3.4"
       - run:
           name: Tag and push Docker images
-          command: .circleci/tag_and_push_docker_image application:$CIRCLE_SHA1
+          command: |
+            .circleci/tag_and_push_docker_image application:$CIRCLE_SHA1 $DSD_DOCKER_REGISTRY/$DSD_DOCKER_IMAGE
+            .circleci/tag_and_push_docker_image application:$CIRCLE_SHA1 $ECR_DOCKER_REGISTRY/$ECR_DOCKER_IMAGE
+  build_nginx:
+    docker:
+      - image: docker:18.05.0-git
```

Comment:
🙋‍♂️ We can use `image: ${ECR_ENDPOINT}/cloud-platform/tools:circleci` here; it is based on the same version of docker and already has `awscli` installed.

Reply:
This comment was thumbed up and resolved but I don't see a change. Should there be a change?

Reply:
I tried it. It didn't work straight off, so switched back.

https://circleci.com/gh/ministryofjustice/fala/133